### ISSUE TITLE
Layer, stats and stats def names should be editable

### ISSUE THREAD

**Issue by sgiaccio** [slms4redd/admin-interface#67]

**Comment by Damianofds** [comment]
mmmm is it really needed? it's a complex task since the resources names are
used as ids in the model and I don't think the effort worth...

2015-06-24 9:59 GMT+02:00 sgiaccio <notifications@github.com>:

**Comment by sgiaccio** [comment]
I agree that it's a complex task and on a production server one could delete the layer and insert it again, but in trainings it's often needed and it's difficult to explain why it's not there...
